Sources/SmokeDynamoDB/DynamoDBTableHistoricalItemExtensions.swift

```diff
@@ -274,40 +276,41 @@ public extension DynamoDBTable {
                                                                              withRetries: retries - 1,
                                                                              completion: completion)
                 } catch {
-                    completion(error)
+                    completion( HTTPResult.error( error ) )
                 }
-            } else {
-                completion(error)
+            } else if let theError = error {
```

Comment:
See my other comment as this will not call `completion` on success.

Reply:
Fixed.